Automattic/wp-calypso
# Plans overhaul Phase 1

#### Changes proposed in this Pull Request

* This is a part of the first phase of the Plan Overhaul project p58i-c1P-p2 .
* With this PR, a new plan `Managed` will be introduced.

#### Testing instructions



Before testing, please make sure:

* You are proxied.
* D75519-code is applied to your sandbox.
* `public-api.wordpress.com` is sandboxed.

Ready? Let's start.

* Signup process
  * Going to `/start/new` as a new user.
  * You should be able to see the new plans in the plans step.
  * <img width="600" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/155700210-98c19d8b-e8de-49a0-b37e-f2563ba6ba8a.png">
  * Check it looks good on small screen as well.
  * Whatever you chose, the process should be completed without any errors.
* Admin `/plans` page when your site is on a free plan.
  * <img width="600" alt="Plans_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/155701008-e499f872-d591-4da3-b5aa-c20ae8ebbe12.png">
  * You should be able to purchase the managed plan with the `Select Managed` button.
* Admin `/plans` page when your site is on the new managed plan.  
  * <img width="600" alt="Plans_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/155701517-47cc8bb4-f651-4a5e-a7bf-b604676782c3.png">
  * Click `View plan` and you will see the purchase information.
  * <img width="600" alt="Purchase_Settings_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/155701740-60d7dcf2-e878-42ce-8239-d578cf62c145.png">
* Admin `/plans` page when your site is on a legacy plan such as Personal, Premium, Business, and eCommerce.
  * Nothing should change.



Related to #
